themes/bootstrap3/templates/Recommend/TopFacets.phtml

```diff
@@ -49,4 +49,4 @@
       <? endif; ?>
     </div>
   <? endforeach; ?>
-<? endif; ?>
+<? endif; ?>
```

Comment:
All these template changes snuck in on master merge somehow. They all need to get reverted. I don't know why they're here.